tomls/omnibus-arbitrum-sepolia-octopus/perps/collaterals/btc.toml

```diff
@@ -0,0 +1,43 @@
+[var.perps_collateral_sBTC_settings]
+btc_upper_limit_discount = "<%= parseEther('0.075') %>"
+btc_lower_limit_discount = "<%= parseEther('0.010') %>"
+btc_discount_scalar = "<%= parseEther('1') %>"
+
+[provision.RewardsDistributor_PerpsCollateral_BTC_liquidation_rewards]
+source = "synthetix-rewards-dist-ext:3.3.21-dev.fd4c5628"
+target = "synthetix-rewards-dist-ext:3.3.21-dev.fd4c5628@BTC_Perps_Rewards"
+options.salt = "<%= settings.salt %>"
+options.rewardManager = "<%= imports.system.contracts.CoreProxy.address %>"
+options.poolId = "<%= extras.spartan_council_pool_id %>"
+options.collateralType = "<%= extras.synth_btc_token_address %>"
+options.payoutToken = "<%= extras.synth_btc_token_address %>"
+options.payoutTokenDecimals = "18"
+options.name = "Perps Collateral Liquidation Rewards for BTC (as BTC)"
+options.authorizedDistributor = "<%= imports.perpsFactory.contracts.PerpsMarketProxy.address %>"
```

Comment:
I think multiple distributors are required, specifically one per core collateral type. Here we configure `collateralType` to be sBTC, however it should reference one of the core collaterals: USDC or WETH.

I actually think this requires 1 distributor per core type (USDC and WETH) and they need to be registered in separate invokes below.